DAO.sol

```diff
@@ -483,8 +487,11 @@ contract DAO is DAOInterface, Token, TokenSale {
     }
 
 
-    function numberOfVotes(uint _proposalID) constant returns (uint _numberOfVotes) {
-        return proposals[_proposalID].votes.length;
+    function unblock() {
```

Comment:
You would have to call it for every address and this gets you back to a loop ....